packages/web-components/src/toolbar/toolbar.spec.md

```diff
@@ -0,0 +1,88 @@
+- @attr size: "small" "medium" "large" | "medium"
+
+### Outputs
+
+### Slots
+
+- label: Content which appears as the label for the toolbar.
+- start: Content which appears at the start of the toolbar, before the default slotted items.
+- default: Content which appears in the default location of the toolbar, left-aligned.
+- end: Content which appears at the end of the toolbar, right aligned.
+
+### CSS Variables
+
+None
+
+## Interactions
+
+Interactions are inherited from FAST Toolbar component.
+
+| Name             | Description                             |
+| ---------------- | --------------------------------------- |
+| `clickHandler`   | Sets the active index                   |
+| `focusinHandler` | Sets focus on element with active index |
+| `keydownHandler` | Handles keydown events on the toolbar   |
+
+## Accessibility
+
+- [x] Find the matching component through WCAG's patterns: https://www.w3.org/WAI/ARIA/apg/patterns/
+  - https://www.w3.org/WAI/ARIA/apg/patterns/toolbar.html
+- [x] Are there any accessibility elements unique to this component?
+- [x] List ARIA attributes
+  - [x] role="toolbar" in FAST
+  - [x] aria-orientation="vertical" in FAST
+- [ ] Does the component support 400% zoom?
+- [ ] What keyboard behaviors does the component support?
+  - [x] Tab : Moves focus to the next focusable element in the toolbar.
+  - [x] Shift + Tab : Moves focus to the previous focusable element in the toolbar.
+  - [x] Arrow keys : Moves focus to the next/previous focusable element in the toolbar.
+
+## Preparation
+
+- [x] Find the base FAST Component
+  - This component will inherit from [FAST Toolbar](https://explore.fast.design/components/fast-toolbar)
+- [x] Check the [Fluent UI React V9 Component Spec](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-toolbar/docs/Spec.md) for differences and document:
+
+  - Differences between FAST Toolbar component and Fluent Toolbar component spec:
+
+    - React Toolbar component contains custom components: `ToolbarButton`, `ToolbarDivider`, `ToolbarGroup`, `ToolbarRadioButton`, `ToolbarRadioGroup`, `ToolbarToggleButton`; whereas FASTToolbar does not.
+    - React Toolbar component has a `size` attribute that can be set to "small", "medium", or "large". FASTToolbar does not have this attribute.
+    - React Toolbar component has a component: `ToolbarItemGroup`. This looks like it is made to contain radio buttons and checkbox elements. FASTToolbar does not have this component.
+
+- [x] Fluent UI React V9 Storybook for implementation differences and document:
+
+  - [Fluent React Toolbar](https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-toolbar--default) has Overflow Items and corresponding More (ellipses) button. This aligns with the Fluent Toolbar spec. The FASTToolbar component however does not implement this. We could either work to include this feature or leave it up to the developer to implement.
```

Comment:
This feature is already in the Fluent monorepo and is a vanilla JS package, there is no dependency on React at all, just simple DOM https://github.com/microsoft/fluentui/tree/master/packages/react-components/priority-overflow

Reply:
@ling1726 Thanks for the added context! I agree. I think this should be added as part of the component.